rfcs/protocols/template-protocol.md

```diff
@@ -59,7 +59,7 @@ variants.
 
 ## Roles
 
-> See [this note](https://github.com/hyperledger/aries-rfcs/tree/master/concepts/0003-protocols/roles-participants-etc.md) for definitions of the terms
+> See [this note](https://github.com/hyperledger/aries-rfcs/tree/main/concepts/0003-protocols/roles-participants-etc.md) for definitions of the terms
```

Comment:
This link is broken. Use https://github.com/hyperledger/aries-rfcs/tree/main/concepts/0003-protocols/#roles-participants-parties-and-controllers instead.